pages/dataset/[datasetid].js

```diff
@@ -35,6 +34,21 @@ const Dataset = ({ dataset }) => {
     })
   }
 
+  /**
+   * Test whether to show a particular section in the metadata field
+   * or not
+   * @param {*} fields
+   */
+  const showSection = (fields) => {
```

Comment:
Makes sense! Thanks for showing me the way. :sunglasses:

Reply:
You're welcome